File: October Dawn 2020/Anupam Food Bill/food_bill.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    int t;
    cin>>t;
    while(t--)
    {
        string s;
        cin>>s;
        int sum = 0;
        string ans = "0";
        for(int i = 0; i<s.length(); i++)
        {
            if(isdigit(s[i]) != 0)
                ans += s[i];
            else{
                if(ans != "0")
                    sum += stoi(ans);
                ans = "0";
            }
        }
        if(ans != "0")
                    sum += stoi(ans);
        cout<<sum<<endl;
    }
    return 0;
}

File: October Dawn 2020/Anupam Food Bill/food_bill.py
t = int(input())
for p in range(t):
    bill = input()
    num = []
    numbers = []
    for i in range(0,len(bill)):
        if bill[i].isdigit():
            num.append(bill[i])
            if i!=len(bill)-1 and not bill[i+1].isdigit():
                ch = ""
                for x in num:
                    ch = ch+x
                numbers.append(ch)
                num = []
            if i+1 == len(bill)-1:
                num.append(bill[i+1])
                ch = ""
                for x in num:
                    ch = ch+x
                numbers.append(ch)
                break
    sum = 0
    for x in numbers:
        sum = sum + int(x)
    print(sum)